kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtWireServiceImpl.java

```diff
@@ -71,22 +71,21 @@
  */
 public final class GwtWireServiceImpl extends OsgiRemoteServiceServlet implements GwtWireService {
 
+    private static final int SERVICE_WAIT_TIMEOUT = 60000;
```

Comment:
The reason is that openJDK is a way slower than the Oracle one. Currently, with 5 seconds that would imply an Exception due to the fact that the wire creation was called with services instances still not running.